### DISCUSSION TITLE
How can I use advanced search to check if an issue is in a project?

### DISCUSSION THREAD

**Discussion by pdgonzalez872** [community/community#83943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Given a search query like `is:open is:issue archived:false org:some_org`, is there a way to check if an issue is being tracked in a project? I'd also like to know the inverse as well: can I search for issues that are NOT being tracked in a project?

Thank you!

**Comment by ghost** [comment]
Search within a Project:-
You can go to the specific project board in your repository and check if an issue is present on that board. This won't be a search query itself, but rather a manual check within the project board.

Use Labels or Milestones:-
Projects often use labels or milestones to categorize issues. You might find issues not being tracked in a project by checking for specific labels or milestones that are exclusive to the project. For instance, if a project-specific label or milestone exists, searching for issues without that label or milestone might help identify issues not tracked in the project.

**Reply by pdgonzalez872** [reply]
Thanks for answering @Yuheshpandian 

> Search within a Project:

Unfortunatel looking at every project and checking if an issue is present doesn't work for us as it is manual and prone to error.

> labels

We thought about this approach. But adding/removing labels (even if they are automatically added on creation) is not optimal either. Maybe we can do some scripting on our end to address this to try to fit labels.

Thanks again.
